frontend/src/screens/apps/NewApp.tsx

```diff
@@ -48,10 +48,22 @@ const NewApp = () => {
     ? budgetRenewalParam
     : "monthly";
 
-  const reqMethodsParam = queryParams.get("request_methods");
-  const [requestMethods, setRequestMethods] = useState(
-    reqMethodsParam ?? Object.keys(nip47MethodDescriptions).join(" ")
+  // returns RequestMethod Set
+  const parseRequestMethods = (reqParam: string): Set<RequestMethodType> => {
```

Comment:
This seems to have broken the edit button, it's no longer visible

Reply:
fixed in https://github.com/getAlby/nostr-wallet-connect/pull/192/commits/4d5f997a45482a78d11c9d781bea1e8675f228b6